src/main/java/com/coveo/pushapiclient/StreamService.java

```diff
@@ -65,7 +65,7 @@ public StreamService(StreamEnabledSource source) {
      * </pre>
      *
      * <p>
-     * For more code samples, visit <a href="TODO: LENS-840">Stream data to your catalog source</a>
```

Comment:
i am sure that this doesnt work :D , since it is a java class in other folder, i checked , this was never done before